db/seeds/users.rb

```diff
@@ -0,0 +1 @@
+User.create(email: 'somedude@gmail.com', user_name: 'somedude', password: 'password', password_confirmation: 'password')
```

Comment:
Line is too long. [120/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.